client/job.lua

```diff
@@ -390,13 +399,16 @@ else
                 rotation = -20,
                 debug = config.debugPoly,
                 onEnter = function()
+                    if QBX.PlayerData.job.type ~= 'ems' then return end
                     local label = QBX.PlayerData.job.onduty and locale('text.onduty_button') or locale('text.offduty_button')
                     lib.showTextUI(label)
                 end,
                 onExit = function()
+                    if QBX.PlayerData.job.type ~= 'ems' then return end
```

Comment:
```suggestion
```
Don't let people get the textui stuck if they lose the EMS job while in the zone